### PR TITLE
WIP: update current state of web console

### PR DESCRIPTION
**Pull request by afshin**
This is the current state of affairs for the web-based version of the Jupyter Qt console: https://github.com/jupyter/jupyter-js-notebook/issues/170

It is a work in progress, this PR is just to keep the development more transparent.

### CODE REVIEW

**Comment by sccolbert** [comment]
Can you post a gif of the current state?

**Comment by afshin** [comment]
screen shot 2016-04-21 at 9 09 39 pm

**Comment by afshin** [comment]
console

File: src/console/widget.ts [hunk 1]
```diff
@@ -142,6 +170,64 @@ class ConsoleWidget extends Widget {
...
+    this._banner = cellsLayout.childAt(0) as RawCellWidget;
+    this._banner.addClass(BANNER_CLASS);
+    this._prompt = cellsLayout.childAt(last) as CodeCellWidget;
+    this._updateBanner();
+  }
+
+  /**
+   * Update the console banner.
+   */
+  private _updateBanner(): void {
```
Comment (blink1073) [inline src/console/widget.ts:221]: Why is this in the view?

**Comment by jasongrout** [comment]
Thanks for posting a screenshot and your WIP.

**Comment by blink1073** [comment]
Merge and iterate!

**Pull request merged**